static/app/views/performance/transactionSummary/transactionReplays/transactionReplays.tsx

```diff
@@ -137,13 +137,22 @@ function ReplaysContent({
   organization: Organization;
   pageLinks: string | null;
 }) {
-  const location = useMemo(() => ({query: {}}) as Location<ReplayListLocationQuery>, []);
+  const location = useLocation();
+
+  if (!eventView.query) {
+    eventView.query = String(location.query.query ?? '');
+  }
```

Comment:
@ryan953 this line fixes the test

Reply:
<img src="https://media2.giphy.com/media/xU9TT471DTGJq/giphy.gif"/>

Reply:
it does nothing in practice, just satisfies the test😭

Reply:
burn it all down, such a pain